concepts/decorators/about.md

```diff
@@ -1,2 +1,44 @@
-#TODO: Add about for this concept.
+# About
 
+Decorators are a feature of the Python language that can modify or register a function.
+
+## How to use them
+
+Decorators are placed just above a function in the code like so:
+
+```py
+@decorator
+def function():
+    pass
+```
+
+Some decorator functions take arguments:

```

Comment:
Are decorators and decorator functions two different things? This discusses decorators then decorator functions without an introduction of the term.